### DISCUSSION TITLE
posts/how-to-setup-sync-google-drive-with-podman-container-and-rclone/

### DISCUSSION THREAD

**Discussion by giscus** [tazihad/tazihad.github.io#6] [Announcements]
# posts/how-to-setup-sync-google-drive-with-podman-container-and-rclone/

Updated guide to sync google drive in linux system with utilising podman container and rclone.

https://zihad.com.bd/posts/how-to-setup-sync-google-drive-with-podman-container-and-rclone/

**Comment by RandomlyHuman** [comment]
Advanced level tutorial :')

**Comment by dadul96** [comment]
Probably a noob question, but what is the advantage of using podman compared to your " Sync Google Drive in Linux using rclone" article from 2023?

I am guessing that the podman approach always uses the latest version of rclone. If that is the case - are you not afraid that updating all the time might introduce some changes that might break your current command set?

**Reply by tazihad** [reply]
Podman should not break commands. Since it uses containers.
I am using Fedora Atomic which has podman preinstalled. That's why I wrote this article.

And there's really no advantage. It's just I prefer using podman container approach.

Thank you for the comment. Did you set up the sync? Let me know if it works for you.

**Reply by dadul96** [reply]
Thanks for the quick response. I don't know much about containers and podman. So i thought the :latest tag will auto-update rclone to the latest available version.

That's why I thought this might be a problem. In case a new rclone version changes for example the arguments for bisync.

But now I get it. Thanks for the nice guide. I'm looking forward to implement this on my system.
